SU2_CFD/src/python_wrapper_structure.cpp

```diff
@@ -31,865 +31,650 @@
-      }
+    
+    int rank = MASTER_NODE;
+    SU2_MPI::Comm_rank(SU2_MPI::GetComm(), &rank);
+    
+    /* --- Initialize boundary conditions customization, this is achieve through the Python wrapper --- */
+    for(iZone=0; iZone < nZone; iZone++){
+        
+        if (config[iZone]->GetnMarker_PyCustom() > 0){
+            
+            if (rank == MASTER_NODE) cout << endl << "----------------- Python Interface Preprocessing ( Zone "<< iZone <<" ) -----------------" << endl;
+            
+            if (rank == MASTER_NODE) cout << "Setting customized boundary conditions for zone " << iZone << endl;
+            for (iMesh = 0; iMesh <= config[iZone]->GetnMGLevels(); iMesh++) {
+                geometry[iZone][INST_0][iMesh]->SetCustomBoundary(config[iZone]);
+            }
+            geometry[iZone][INST_0][MESH_0]->UpdateCustomBoundaryConditions(geometry[iZone][INST_0], config[iZone]);
+            
+            if ((config[iZone]->GetKind_Solver() == MAIN_SOLVER::EULER) ||
+                (config[iZone]->GetKind_Solver() == MAIN_SOLVER::NAVIER_STOKES) ||
+                (config[iZone]->GetKind_Solver() == MAIN_SOLVER::RANS) ||
+                (config[iZone]->GetKind_Solver() == MAIN_SOLVER::INC_EULER) ||
+                (config[iZone]->GetKind_Solver() == MAIN_SOLVER::INC_NAVIER_STOKES) ||
+                (config[iZone]->GetKind_Solver() == MAIN_SOLVER::INC_RANS) ||
+                (config[iZone]->GetKind_Solver() == MAIN_SOLVER::NEMO_EULER) ||
+                (config[iZone]->GetKind_Solver() == MAIN_SOLVER::NEMO_NAVIER_STOKES)) {
+                
+                solver[iZone][INST_0][MESH_0][FLOW_SOL]->UpdateCustomBoundaryConditions(geometry[iZone][INST_0], config[iZone]);
+            }
+        }
     }
-  }
-
+    
 }
 
 /////////////////////////////////////////////////////////////////////////////
 /* Functions related to the global performance indices (Lift, Drag, ecc..) */
 /////////////////////////////////////////////////////////////////////////////
 
 passivedouble CDriver::Get_Drag() const {
-
-  unsigned short val_iZone = ZONE_0;
-  unsigned short FinestMesh = config_container[val_iZone]->GetFinestMesh();
-  su2double CDrag, factor, val_Drag;
-
-  /*--- Calculate drag force based on drag coefficient ---*/
-  factor = solver_container[val_iZone][INST_0][FinestMesh][FLOW_SOL]->GetAeroCoeffsReferenceForce();
-  CDrag = solver_container[val_iZone][INST_0][FinestMesh][FLOW_SOL]->GetTotal_CD();
-
-  val_Drag = CDrag*factor;
-
-  return SU2_TYPE::GetValue(val_Drag);
+    
+    unsigned short val_iZone = ZONE_0;
+    unsigned short FinestMesh = config_container[val_iZone]->GetFinestMesh();
+    su2double CDrag, factor, val_Drag;
+    
+    /*--- Calculate drag force based on drag coefficient ---*/
+    factor = solver_container[val_iZone][INST_0][FinestMesh][FLOW_SOL]->GetAeroCoeffsReferenceForce();
+    CDrag = solver_container[val_iZone][INST_0][FinestMesh][FLOW_SOL]->GetTotal_CD();
+    
+    val_Drag = CDrag*factor;
```

Comment:
Fix this file please, or run clang-format it is difficult to see changes like this.